addons/l10n_syscohada/models/account_move.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from odoo import models
+
+
+class AccountMoveLine(models.Model):
+    _name = "account.move.line"
+    _inherit = ["account.move.line"]
```

Comment:
```suggestion
    _inherit = "account.move.line"
```
If you're not changing the `_name`, you don't need to specify it.
And if you're only inheriting from one model, you don't need to put it in a list :wink: